lib/ai/tools/terminal-agent.ts

```
@@ -95,74 +109,112 @@ export async function executeTerminalAgent({
+            isPremiumUser,
+            agentMode,
+          }),
+          maxSteps: 5,
+          toolChoice: 'required',
+          abortSignal,
+          onFinish: async ({ finishReason }: { finishReason: string }) => {
+            if (supabase) {
+              await handleChatWithMetadata({
+                supabase,
+                chatMetadata: config.chatMetadata!,
+                profile,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)